### ISSUE TITLE
Inconsistent separator handling

### ISSUE THREAD

**Issue by te1** [kevinongko/vue-numeric#35]
Handling of decimal separator doesn't quite work if you want to use a comma instead of a dot.

Steps to reproduce
- Go to https://kevinongko.github.io/vue-numeric
- Select "use '.' as separator" (this is the thousand separator)
- Enter Value "1,2" (with a comma) and leave (blur) input
- Focus input, it now says "1.2" (with a dot) but should be "1,2" (with a comma)
- Add 3 to the input and leave (blur) input
- Value is now 123.0 but should be 1.23

I got it to work by changing the the following
```js
onFocusHandler () {
  this.amount = accounting.formatNumber(this.valueNumber, {
    precision: Number(this.precision),
    decimal: this.decimalSeparator,
    thousand: ''
  })
}
```
but this breaks #26

**Comment by te1** [comment]
It may not be the most elegant solution but this works for me without negative side effects so far:

```js
onFocusHandler () {
  this.amount = this.valueNumber.toString().replace('.', this.decimalSeparator)
}
```

**Comment by wagnereliakim** [comment]
@te1 I found the same problem.
Your las solution works fine, but I  think that we can simplify by calling the existing function for this.

```javascript
onFocusHandler () {
  this.amount = this.format(this.valueNumber)
}
```

**Comment by kevinongko** [comment]
Should be fixed in 2.2.3
